cpp/README.md

```diff
@@ -110,8 +110,8 @@ ASAN, and `ARROW_USE_ASAN` is mutually-exclusive with the valgrind option
 ### Building/Running fuzzers
 
 Fuzzers can help finding unhandled exceptions and problems with untrusted input that
-may lead to crashes, security issues and undefined behavior. They do this by
-generating random input data and observing the behavior of the executed code. To build
```

Comment:
"behavior" appears to be the preferred spelling in the US http://grammarist.com/spelling/behavior-behaviour/

Reply:
Agree, please revert